lib/src/block_parser.dart

```diff
@@ -35,6 +35,18 @@ class BlockParser {
   /// Index of the current line.
   int _pos = 0;
 
+  /// Starting line of the last unconsumed content.
+  int _start = 0;
+
+  /// The lines from [_start] to [_pos] (inclusive), it works as a buffer for
+  /// some blocks, for example:
+  /// When the [ParagraphSyntax] parsing process is interrupted by the
+  /// [SetextHeaderSyntax], so this structure is not a paragraph but a setext
+  /// heading, then the [ParagraphSyntax] does not have to retreat the reading
+  /// position, it only needs to return `null`, the [SetextHeaderSyntax] will
```

Comment:
Let's mention the method name here, something like "then the [ParagraphSyntax.parse] method does not..."

Reply:
Cool, we support this [A.B] format now.